src/Sylius/Bundle/CartBundle/Resources/translations/messages.en.yml

```diff
@@ -0,0 +1,4 @@
+sylius:
+    form:
+        cart_item:
+            quantity: Quantity
```

Comment:
There are two blank lines at the end now :) If you want to add a new line at the end, the diff would be `-0 +1` instead of `-1 +2`.